.github/workflows/push_pr.yml

```diff
@@ -29,3 +29,11 @@ jobs:
     uses: apache/flink-connector-shared-utils/.github/workflows/ci.yml@ci_utils
     with:
       flink_version: ${{ matrix.flink }}
+    
+  python_test:
+    strategy:
+      matrix:
+        flink: [ 1.17.1, 1.18.0 ]
+    uses: pvary/flink-connector-shared-utils/.github/workflows/python_ci.yml@ci_utils
```

Comment:
This should be changed to `apache/flink-connector-shared-utils/.github/workflows/python_ci.yml@ci_utils` once https://github.com/apache/flink-connector-shared-utils/pull/27 merged